Bugzilla/User.pm

```diff
@@ -448,6 +456,10 @@ sub _check_numeric {
   return $value;
 }
 
+sub _check_timestamp {
+  return Bugzilla->dbh->selectrow_array('SELECT LOCALTIMESTAMP(0)');
```

Comment:
Question: why `LOCALTIMESTAMP` instead of `UNIX_TIMESTAMP` here?

Reply:
I am only using the unix timestamps for the migration part as it is easier to determine which is the larger value and use that for the modifcation_ts. For normal updating of users using ->update() we always want the timestamp in the standard `YYYY-MM-DD HH:MI:SS` format which is what we get from LOCALTIMESTAMP.